release/default.nix

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+{ pkgs ? import (import ../nix/sources.nix).nixpkgs { } }:
+let
+  executable = (import ../crossref-verifier.nix {
+    static = true;
+  }).components.exes.crossref-verify;
+  joinByBasename = name: paths:
+    pkgs.runCommandNoCC name { } (''
+      mkdir $out
+    '' + pkgs.lib.concatMapStrings (path: ''
+      ln -s ${path} $out/${builtins.baseNameOf path}
+    '') paths);
+in joinByBasename "crossref-verifier-release" [
+  "${executable}/bin"
+  ../LICENSES
+  ../README.md
```

Comment:
README is definitely redundant in the archive, everyone will use the github page to learn how to download and use crossref-verifier

Reply:
Maybe. Can @gromakovsky and @Martoon-00 express their opinion about this?

Reply:
I don't know how it is usually done, though assuming that users have read README at GitHub upon getting the archive seems like a sane assumption to me.

Reply:
> I don't know how it is usually done

Same here. I think I've seen several repos where releases contain documentation files (README and others present in the repo), but I don't know the reasoning behind that. E. g. `stack` releases seem to contain `README.md` and some other docs. It seems to me that putting `README.md` is useful only if we want to put a link to this archive to some website.

So, on the one hand `README.md` seems to be useless currently, but on the other hand it makes the archive more self-contained and suitable for other ways of distribution. Also it seems to be quite harmless: if we have to put licenses (which is being discussed in another conversation here), it should be an archive anyway and one `README.md` file is negligible. Though if we conclude that including licenses is not necessary, we'll be able to put only the binary into releases and then people won't have to extract it.